Makefile

```diff
@@ -24,6 +24,12 @@ fioctl-%:
 		go build $(LDFLAGS) -o bin/$@ main.go
 	@if [ "$@" = "fioctl-windows-amd64" ]; then mv bin/$@ bin/$@.exe; fi
 
+fioctl-linux-armv7:
```

Comment:
I would wait a bit for Volodya his work on the xgo stuff. 😊

Reply:
This PR is all about adding some tests. Volodya needs to build and test manually (very time consuming see;`I need a volunteer to test the binaries on Mac OS and Arm Linux.` ) what here has been done here automatically (all less mac/arm64);

Therefore, I’m thinking this PR could really speed things up and bring confidence regards his changes! So, I’m kinda wondering, why wait? How could this PR negatively impact this effort?

Reply:
My first thought was we use the check to ensure the xgo stuff but it is totally fine.

Reply:
we've agreed in a different PR to drop armv7. We've never included it in a release artifact.

Reply:
We can just remove this build/check